unittests/finalizer_update_tests.cpp

```diff
@@ -52,10 +52,10 @@ BOOST_AUTO_TEST_CASE(savanna_set_finalizer_single_test) { try {
    t.produce_block();
    t.check_head_finalizer_policy(1u, pubkeys0); // new policy should only be active until after two 3-chains
 
-   t.produce_blocks(3);
+   t.produce_blocks(num_chains_to_final);
    t.check_head_finalizer_policy(1u, pubkeys0); // one 3-chain - new policy still should not be active
 
-   t.produce_blocks(2);
+   t.produce_blocks(num_chains_to_final-1);
    t.check_head_finalizer_policy(1u, pubkeys0); // one 3-chain + 2 blocks - new policy still should not be active
```

Comment:
comment is wrong, it should be `+ 1 block`

Reply:
Fixed it by https://github.com/AntelopeIO/spring/pull/389/commits/c1feb767a7ede694db9dae81d0af3ccf57721936

Thanks. I am updating comments in other files.